src/app-pages/map/components/map-details/components/DamProfile.js

```diff
@@ -19,6 +20,38 @@ export const damProfileKeys = [
   "current_surcharge",
 ];
 
+// /** @type a2w.models.DamProfileHistoryEntry[] */
+// const damD3ChartKeys = [
+//   "date",
+//   "in_flow",
+//   "top_of_dam",
+//   "top_of_flood",
+//   "bottom_of_flood",
+//   "stream_bed",
+//   "out_flow",
+// ];
+// Using dummy default data for now. Will put additonal logic in for mode and horizontalLabels. 
+const data = {
```

Comment:
I think it would be better to initialize the dummy data here for now. I'm guessing once we have the real data, the DamProfile component will create the data object and pass it to the DamProfileChart, the DamProfileChart shouldn't have to set the data itself. So here you can do
```
const data = {
  mode: "lockTurbine", // mode could be lock or dam or lockTurbine or turbine. Create func to calc
  hasLock: true,
  hasTurbine: true,
  damTop: 1000,
  damBottom: 500,
  horizontalLabels: [
    { name: "Top of Dam", value: 1000, showLine: true, side: "left"},
    { name: "Top of Flood", value: 950, showLine: true, side: "left"},
    { name: "Top of Conservation", value: 790, showLine: true, side: "left" },
  ],
  currentLevel: 800,
  tailWater: 600,
  inflow: 100,
  outflow: 200,
  sur: 10,
  text: "Dam",
  date: 1571174251, // 15 Oct 2019, 16:17:31
};
```
and the DamProfileChart just needs to dereference all of the keys.

Reply:
Yeah, that makes sense. I intended line 1100 in the chart to set default data. Pushed up the adjustment.

Reply:
Ah gotcha